graph/schema.graphqls

```diff
@@ -36,7 +43,7 @@ type Query {
   ): AftermarketDeviceConnection!
   vehicle(id: Int!): Vehicle!
 
-  dcn(node: Bytes!): DCN!
+  dcn(input: DCNInput!): DCN!
```

Comment:
I like the naming from [the RFC](https://github.com/graphql/graphql-spec/pull/825). `dcn(by: DCNBy!)`.